project/src/components/place-card/place-card.tsx

```diff
@@ -2,36 +2,39 @@ import Badge from '../badge/badge';
 import Bookmark from '../bookmark/bookmark';
 import {Link} from 'react-router-dom';
 import {AppRoute} from '../../const/routing';
+import { MULTIPLER } from '../../const/general';
+import {Offer} from '../../types/offers';
 
 type PlaceCardProps = {
   className: string;
+  offer: Offer;
 }
 
-function PlaceCard({className}: PlaceCardProps): JSX.Element {
+function PlaceCard({className, offer}: PlaceCardProps): JSX.Element {
   return (
     <article className={`${className} place-card`}>
-      <Badge className="place-card__mark" />
+      {offer.isPremium ? <Badge className="place-card__mark" /> : ''}
```

Comment:
В реакте есть conditional rendering, давай договоримся что такие конструкции мы не используем) 
```suggestion
      {offer.isPremium && <Badge className="place-card__mark" />}
```